locations/fblipke.yml

```diff
@@ -0,0 +1,75 @@
+---
+location: fblipke
+location_nice: FBLipke - Freifunk Berlin
+latitude: 52.484948320
+longitude: 13.443380903
+altitude: 42
+height: 13
+contact_name: 'Guido L.'
+contact_nickname: 'LipkeGu'
+contacts:
+  - 'contact@fblipke.de'
+hosts:
+  - hostname: fblipke-core
+    role: corerouter
+    model: "x86_64"
+    wireless_profile: freifunk_default
+    low_flash: true
+  - hostname: fblipke-ap01
+    role: ap
+    model: "ubnt_nanostation_m2_xw"
+    wireless_profile: freifunk_default
+    low_flash: true
+
+ipv6_prefix: "2001:bf7:850:1500::/56"
+
+networks:
+  - vid: 20
+    role: mesh
+    name: 11s_n_2g
+    prefix: 10.248.2.200/32
+    ipv6_subprefix: -20
+    mesh_ap: fblipke-core
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+  - vid: 40
+    role: dhcp
+    prefix: 10.248.3.128/27
+    ipv6_subprefix: 0
+    inbound_filtering: true
+    enforce_client_isolation: true
+    no_corerouter_dns_record: true
+    assignments:
+      fblipke-core: 129
+      fblipke-ap01: 130
```

Comment:
You don't need this line. The ap will be reachable via the ip of the management network (vid 42).